src/main/scala/org/globalforestwatch/layers/DeadwoodCarbon2000.scala

```diff
@@ -5,6 +5,7 @@ import org.globalforestwatch.grids.GridTile
 case class DeadwoodCarbon2000(gridTile: GridTile, model: String="standard")
   extends FloatLayer
     with OptionalFLayer {
-  val model_suffix = if (model == "standard") "" else s"__$model"
-  val uri: String = s"$basePath/gfw_deadwood_carbon_stock_2000$model_suffix/v20191106/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/Mg/geotiff/${gridTile.tileId}.tif"
```

Comment:
Need to update layer path in datalake to "$basePath/gfw_deadwood_carbon_stock_2000$model_suffix/v20191106/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/Mg_ha-1/geotiff/${gridTile.tileId}.tif"